core/utils.py

```diff
@@ -24,19 +42,152 @@ def load_mrsa_data():
+    addresses = dict(zip(cms_metric_df['Address'].values,cms_metric_df['Address'].index))
+
+    if place_address in addresses:
+        match_ = cms_metric_df.loc[addresses[place_address],:]
+        print(f"address match: '{place_address}' with '{match_['Address']}'")
+        return match_
+    else:
+        # Try to find a close match if there was no exact match
+        matches = get_close_matches(place_address, addresses.keys(), n=1, cutoff=0.9)
+        if matches:
+            match_ = cms_metric_df.loc[addresses[matches[0]],:]
+            print(f"address match: '{place_address}' with '{match_['Address']}'")
+            return match_
+
+    #return empty df if no hits
+    return pd.DataFrame()
+
+@timeit
+def format_address(place_address: str) -> str:
+    """ Formats the address to match the data set"""
+    print("place_address preformat: ", place_address)
+    # Google map returns addresses with abbreviations, the data set has full names
+    place_address = place_address.lower().replace("-", " ").replace("/", " ")
+    place_address = place_address.replace(' ave ', ' avenue ')
+    place_address = place_address.replace(' rd ', ' road ')
+    place_address = place_address.replace(' st ', ' street ')
+    place_address = place_address.replace(' dr ', ' drive ')
+    place_address = place_address.replace(' ct ', ' court ')
+    place_address = place_address.replace(' e ', ' east ')
+    place_address = place_address.replace(' w ', ' west ')
+    place_address = place_address.replace(' n ', ' north ')
+    place_address = place_address.replace(' s ', ' south ')
+    place_address = place_address.replace(' ne ', ' northeast ')
+    place_address = place_address.replace(' nw ', ' northwest ')
+    place_address = place_address.replace(' se ', ' southeast ')
+    place_address = place_address.replace(' sw ', ' southwest ')
+    place_address = place_address.replace(' blvd ', ' boulevard ')
+    place_address = place_address.replace(' pkwy ', ' parkway ')
+    place_address = place_address.replace(' pl ', ' place ')
+    place_address = place_address.replace(' ln ', ' lane ')
+    place_address = place_address.replace(' hwy ', ' highway ')
+    place_address = place_address.strip()
+    place_address = place_address.replace(", united states", "")
+
+    split_address = place_address.split(" ")
+    final_address = []
+    for word in split_address:
+        n = parse_ordinal(word)
+        if n and not word.isnumeric():
+            word = ordinal(n)
+        final_address.append(word)
+
+    place_address = " ".join(final_address)
+    print("place_address formatted: ", place_address)
+    return place_address
+
+def ordinal(n: int) -> str:
+    if 11 <= (n % 100) <= 13:
+        suffix = 'th'
+    else:
+        suffix = ['th', 'st', 'nd', 'rd', 'th'][min(n % 10, 4)]
+    return str(n) + suffix
+
+### Google Maps API
+@timeit
+def get_places(google_maps: GoogleMapsClient, gmaps_places_args: Dict, radius: int = 10000) -> Dict:
```

Comment:
was this worth breaking into its own function?

Reply:
Only for understanding load time, otherwise no it wasn't. The API call takes ~0.3 sec give or take and all the parsing takes 3-5 sec.